a3p-integration/proposals/p:upgrade-19/replaceFeeDistributor.test.js

```diff
@@ -166,7 +166,7 @@ test('replace feeDistributor', async t => {
   );
 
   for await (const i of asyncGenerator(vaultCount)) {
-    console.log('----- Starting sequence -----', i);
+    t.log('----- Starting sequence -----', i);
```

Comment:
these aren't equivalent. Usually we've used console for messages in which you want to see the interleaving with other console messages. I don't know whether that's the case here.

Reply:
Yeah, on closer inspection this seems to fit that pattern.